src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Rules/AssemblyIdentityMustMatchTests.cs

```diff
@@ -202,57 +144,54 @@ public void LeftAssemblyKeyTokenNull(bool strictMode)
             }
         }
 
-        [RequiresMSBuildVersionTheory("17.0.0.32901")]
+        [Theory]
         [InlineData(true)]
         [InlineData(false)]
         public void RightAssemblyKeyTokenNull(bool strictMode)
         {
-            var testAsset = _testAssetsManager
-                .CopyTestAsset("PublicKeyTokenValidation", allowCopyIfPresent: true)
```

Comment:
can you delete this asset as well from the assets folder

Reply:
Ah, yeah I forgot to do that. Thanks 😄